### PR TITLE
Firefly-1237: fixed

### PR DESCRIPTION
**Pull request by robyww**
#### [Firefly-1237](https://jira.ipac.caltech.edu/browse/FIREFLY-1237)
 
 - _Note_- There was a race condition so I used `setTimeout`, in the future it would be worth looking at a bigger solution.

#### Testing
  - https://firefly-1237-polygon.irsakudev.ipac.caltech.edu/irsaviewer
  - test according to the ticket

### CODE REVIEW

File: src/firefly/js/ui/CatalogSearchMethodType.jsx [hunk 1]
```diff
@@ -280,7 +280,7 @@ function SizeArea({groupKey, searchType, imageCornerCalc}) {
             if (!cornerCalcV) {
                 if (sel) setVal('imageCornerCalc', 'area-selection');
             }
-            setVal('polygoncoords', calcCornerString(pv, cornerCalcV));
+            setTimeout( () => setVal('polygoncoords', calcCornerString(pv, cornerCalcV)), 4);
```
Comment (loitly) [inline src/firefly/js/ui/CatalogSearchMethodType.jsx:283]: Curious... if only a 4ms delay was needed, shouldn't a no delay work as well?

**Review by loitly — APPROVED** [review]
Code change looks fine.

File: src/firefly/js/ui/CatalogSearchMethodType.jsx (hunk 1 above)
Comment (robyww) [reply src/firefly/js/ui/CatalogSearchMethodType.jsx:283]: probably

**Comment by lrebull** [comment]
i confirm it works!

**Pull request merged**